Sources/MongoSwift/MongoClient.swift

```diff
@@ -357,9 +358,10 @@ public class MongoClient {
      */
     public func listDatabases(
         _ filter: Document? = nil,
+        _ options: ListDatabasesOptions = ListDatabasesOptions(),
         session: ClientSession? = nil
     ) -> EventLoopFuture<[DatabaseSpecification]> {
-        let operation = ListDatabasesOperation(client: self, filter: filter, nameOnly: nil)
+        let operation = ListDatabasesOperation(client: self, filter: filter, nameOnly: false, options: options)
```

Comment:
I think `nameOnly` can stay `nil` here. the idea behind that is we just don't send the flag at all when we don't want `nameOnly`.